win32ss/user/ntuser/winpos.c

```diff
@@ -1393,9 +1393,16 @@ WinPosDoOwnedPopups(PWND Window, HWND hWndInsertAfter)
 
                if (List[i] == Owner)
                {
-                  if (i > 0) hWndInsertAfter = List[i-1];
-                  else hWndInsertAfter = topmost ? HWND_TOPMOST : HWND_TOP;
-                  break;
+                   if (i > 0)
+                   {
+                       if (List[i - 1] != Window->head.h)  
+                           hWndInsertAfter = List[i - 1];
```

Comment:
So, what happens when `List[i - 1] == Window->head.h` instead? What does this mean?
Also, adding comments to explain **what's the reasoning** behind what is being done there, would be welcome.

Reply:
Added comments for explanation.